test/powershell/Language/Scripting/ColorizePrompt.Tests.ps1

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+Describe "Colorizing prompt" -Tags 'CI', 'Slow' {
+    Context 'Prompt can be colorized' {
+        It 'Prompt can be colorized' {
+            $colors = @'
+            $PSStyle.Prompt.Caption = "`e[34m"
+            $PSStyle.Prompt.Message = "`e[92m"
+            $PSStyle.Prompt.Help = "`e[95m"
+            $PSStyle.Prompt.ChoiceDefault = "`e[41m"
+            $PSStyle.Prompt.ChoiceOther = "`e[106m"
+            $PSStyle.Prompt.ChoiceHelp = "`e[47m"
+'@
+
+            # Only reasonable way to force "interactivity" and capture prompts is to fork
```

Comment:
please take a look at the custom host, which may solve your problem here (and potentially speed up your tests). You can find the module here: `test/tools/Modules/HelpersHostCS/HelpersHostCS.psm1` and an example may be found at: `test/powershell/Modules/Microsoft.PowerShell.Utility/Read-Host.Tests.ps1`

Reply:
nope - it doesn't help. The test host would need the psstyle changes to be implemented in it as well.

Reply:
Apologies, I'm not sure whether this means I should do something or not. To be frank, this PR has been stuck waiting for merge for over a year, so if you want me to add a bunch of code to test it, I'll just mark it as untestable.

Reply:
to be clear, i think this is an interactive only scenario test.

Reply:
Sure, what I have done is quite a dirty and fragile hack. You want me to remove this test?